src/device.rs

```diff
@@ -48,6 +48,26 @@ pub unsafe extern "C" fn wgpuAdapterRequestDevice(
     (callback.unwrap())(device_id, userdata);
 }
 
+#[no_mangle]
+pub unsafe extern "C" fn wgpuAdapterGetProperties(
```

Comment:
We shouldn't add arbitrary entry points to the library, this will break compatibility with webgpu-headers. Instead these extra things should be passed as the extension structures to the existing methods.

Reply:
https://github.com/webgpu-native/webgpu-headers/blob/8b8c2a77dd5a94a64d987b664e2ec320e34a3dc5/webgpu.h#L1074 ;)

Reply:
oh well, thanks for the reference :sweat_smile: